bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestCommandWorkItem.java

```diff
@@ -146,6 +147,23 @@ private Optional<Hash> resultingCommitHash(List<Comment> allComments) {
                  .findAny();
     }
 
+    private void changeLabelsAfterComment(List<String> labelsToAdd, List<String> labelsToRemove){
+        if (labelsToAdd != null && !labelsToAdd.isEmpty()) {
+            for (var label : labelsToAdd) {
+                if (!pr.labelNames().contains(label)) {
+                    pr.addLabel(label);
```

Comment:
Could you add an info log about adding and removing labels to the PR.

Reply:
Fixed.